### ISSUE TITLE
Feature Request - Return true 

### ISSUE THREAD

**Issue by azme94** [replaysMike/AnyDiff#24]
Request: A method that returns immediately, at the first encounter of difference, without any further difference checks. 
bool IsDiff(ob1, obj2)

Justification: Sometimes, the app only cares about if the objects are same or not and then only write the updated full object to DB.

**Comment by replaysMike** [comment]
Good idea, I'll look into implementing that on Monday as it would be pretty easy.

I may even through in some performance improvements since I haven't touched this lib in a while
